### PR TITLE
Support "rasa run actions" with actions as dir

### PR DESCRIPTION
**Pull request by tjpalmer**
**Proposed changes**:
- Support "rasa run actions" with an actions directory, per issue #3865

**Status (please check what you already did)**:
- [X] made PR ready for code review
- [X] added some tests for the functionality
- [X] updated the documentation
- [X] updated the changelog
- [X] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by erohmensing** [comment]
Thanks for the PR, we'll give it a review as soon as possible.

File: rasa/cli/run.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def add_subparser(
     )
     run_parser.set_defaults(func=run)
 
+    # NOTE: subparsers here need special handling in
```
Comment (tabergma) [inline rasa/cli/run.py]: I think we can remove the note here. Instead I would add a comment to `parse_last_positional_argument_as_model_path`.

Comment (tjpalmer) [reply rasa/cli/run.py]: I changed comments per your recommendation. Hopefully I got what you were looking for.

**Comment by tjpalmer** [comment]
If travis passes, this might be good to go. Let me know if I need to update anything.

**Pull request merged**